### ISSUE TITLE
5.6. I Monitor

### ISSUE THREAD

**Issue by JirkaDellOro** [JirkaDellOro/Computergrafik.Online#266]
So ist die Interaktion völlig unsinnig und überflüssig!

**Comment by xcluu** [comment]
Durch die Drehbuchvorgaben wird der richtige Sinn der Interaktion leider nicht ersichtlich... Hast du hierzu vielleicht einen konkreteren Verbesserungsvorschlag?

**Comment by JirkaDellOro** [comment]
Klar, Es geht ja darum die Subpixel zu sehen. Der Monitor, der im Drehbuch gemeint ist, befindet sich auf des Roboters Bauch. Dann weiß man auch, dass es jetzt nicht um das Bild geht, sondern um den Monitor und wie die Pixel aussehen. Zoom man hoch, sieht man etwas wie hier bei LCD: https://en.wikipedia.org/wiki/Subpixel_rendering#/media/File:Pixel_geometry_01_Pengo.jpg
